docs/core/compatibility/toc.yml

```diff
@@ -32,7 +32,7 @@
     href: corefx.md
   - name: Cryptography
     href: cryptography.md
-  - name: Entity Framework Core
+  - name: Entity Framework Core >>
```

Comment:
```suggestion
  - name: Entity Framework Core
```

Reply:
I put those there to signify that you're leaving this TOC. Another option is to create a contextual link where you remain in this TOC?

Reply:
I don't think we have a standard way to indicate that you're leaving the TOC.  There was a team discussion about this some time ago that so far as I recall was inconclusive.  If not everybody is using chevrons to indicate that, it will be confusing rather than helpful.  The contextual toc links are possible, but they have bugs and after implementing them in the ASP.NET Core doc set I would say they were not worth the effort.  I would just leave the link as-is, as in the @scotaddie suggested change.

Reply:
What about moving EF Core to the end of the list then?

Reply:
That seems fine to me.

Reply:
In my opinion, it's best to maintain the alphabetical ordering. It makes scanning the list a bit quicker.

Reply:
I like it at the end, because at that point if you leave the TOC, you're already "finished" with the current TOC anyway. But I'll go with popular decision on this one.

Reply:
yeah, I think I prefer the alphabetical order too. Also, would we be able to use contextual TOC here? Maybe you didn't even have to leave if we could.